### ISSUE TITLE
Dynamic .env file

### ISSUE THREAD

**Issue by g5becks** [go-task/task#434]
This may be possible already and I am unaware of it, but what I would like to be able to do is set the dotenv file based on an environment variable.
E.G.

```
version: "3"

env:
  NODE_ENV: echo $($NODE_ENV)

dotenv: ["./packages/server/.env.{{.NODE_ENV}}"]
```
This would enable more dynamic variables for my tasks as opposed to writing separate taskfiles and including them.

Thanks.

**Comment by badele** [comment]
In the meantime, you can use this workaround

```
# I assume that NODE_ENV is in shell variable 
set -o allexport; source ./packages/server/.env.${NODE_ENV}; set +o allexport
task target
```

**Comment by badele** [comment]
The best feature would be have a taskfile to be able to use variables on any **yaml** section (from `vars`, `env` or minima environment vars)

**Comment by andreynering** [comment]
This was just implemented at 08265ed1d7833bf3bbc4d0f060f7e3277e6f1d3e.
